### PR TITLE
docs(readme): Updating readme with more content

### PR DESCRIPTION
**Pull request by JordanSussman**
## Proposed change

This is an initial first stab at updating the readme. I'd ideally like to add a gif/video showcasing the ease of use, but that can come later.

## Types of changes

What types of changes is this pull request introducing to flottbot? _Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

You can fill this out after creating your PR. _Put an `x` in the boxes that apply_

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,31 +1,64 @@
...
+[![docker](https://img.shields.io/docker/automated/target/flottbot.svg)](https://hub.docker.com/r/target/flottbot)
 
-A chatbot framework written in Go. But there's a catch, you don't need to know a lick of Go! Configure your bot via YAML files, extend functionality by writing scripts in your favorite language.
+Flottbot is a chatbot framework written in Go. But there's a catch, you don't need to know a lick of Go! Configure your bot via YAML files, extend functionality by writing scripts in your favorite language.
 
 The philosophy behind flottbot is to create very simple, lightweight, "dumb" bots that interact with APIs and scripts which house a bot's business logic. The word **flott** comes from the German word meaning _quick_/_speedy_.
 
+1. [Installation](#installation)
+1. [Docker images](#docker-images)
+1. [Available providers](#available-providers)
```
Comment (JordanSussman) [inline README.md]: I feel like providers may not be the best wording for this section.... Any thoughts?

File: README.md [hunk 2]
```diff
@@ -1,31 +1,64 @@
-# flottbot
+<p align="center">
+  <img alt="flottbot logo" src="https://i.imgur.com/P9NI00w.png" height="160" />
```
Comment (JordanSussman) [inline README.md]: Should we make this larger?

File: README.md (hunk 1 above)
Comment (sjqnn) [reply README.md]: Maybe stick to the codebase terminology of `remotes`?

File: README.md (hunk 2 above)
Comment (sjqnn) [reply README.md]: looks good for now

**Pull request merged**